ark/type/methods/base.ts

```diff
@@ -42,12 +42,25 @@ import type { instantiateType } from "./instantiate.ts"
 interface Type<out t = unknown, $ = {}>
 	extends Callable<(data: unknown) => distill.Out<t> | ArkErrors> {
 	[inferred]: t
+	/**
+	 * Generic argument. May be used for type reconstruction.
+	 * @example type A = type.infer<[typeof T.t, '[]']>
+	 */
 	t: t
+	/**
+	 * Result type of the validation. Base type used by
```

Comment:
Probably this should be:

"A type representing the output the Type will return (after morphs are applied to valid input)"